src/App.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <v-app id="app">
-    <sidebar />
-    <router-view :apps='apps' />
+    <!-- TODO: REMOVE THIS REF WHEN VUEX IS ADDED -->
+    <sidebar ref="sidebar" />
+    <!-- TODO: REMOVE THIS UPDATE WHEN VUEX IS ADDED -->
```

Comment:
Could you open an issue for these and link the lines of code in the issue description?